apps/optimizely/src/optimizely-client.js

```diff
@@ -25,39 +25,64 @@ export default class OptimizelyClient {
     this.onReauth();
   };
 
-  getProjects() {
-    return this.makeRequest('/projects');
-  }
-
-  getExperiment = (experimentId) => {
-    return this.makeRequest(`/experiments/${experimentId}`);
-  };
-
-  _getExperimentsPerPage = (perPage, page) => {
-    return this.makeRequest(
-      `/experiments?project_id=${
-        this.project
-      }&per_page=${perPage.toString()}&page=${page.toString()}`
-    );
-  };
-
-  getExperiments = async () => {
-    let experiments = [];
+  _getItemsPerPage = async (item) => {
+    let items = [];
     const PER_PAGE = 100;
-    const MAX_REQUESTS = 5;
+    const MAX_REQUESTS = 10;
 
     for (let i = 1; i <= MAX_REQUESTS; i++) {
-      const results = await this._getExperimentsPerPage(PER_PAGE, i);
-      experiments = [...experiments, ...results];
+      const results = await this.makeRequest(this._getItemsUrl(PER_PAGE, i, item));
+      items = [...items, ...results];
       if (results.length < PER_PAGE) {
         break;
       }
     }
 
-    experiments = experiments.filter((experiment) => {
-      return experiment.status !== 'archived';
+    if (item === 'experiment') {
+      items = items.filter((experiment) => {
+        return experiment.status !== 'archived';
+      });
+    }
+
+    items = items.sort((a, b) => {
+      const nameA = a.name.toUpperCase();
+      const nameB = b.name.toUpperCase();
+
+      if (nameA < nameB) {
+        return -1;
+      }
+      if (nameA > nameB) {
+        return 1;
+      }
+      return 0;
     });
```

Comment:
there might be a lodash function that handle this (`sortBy`?) but not required at all.

Reply:
Thanks for the suggestion! I'll keep the current implementation, but good to remember for the future!